tce/src/applibs/LLVMBackend/plugin/TCEISelLowering.cc

```diff
@@ -1333,6 +1336,87 @@ TCETargetLowering::LowerShift(SDValue op, SelectionDAG& dag) const {
     return op;
 }
 
+SDValue
+TCETargetLowering::lowerHWLoops(SDValue op, SelectionDAG& dag) const {
+    if (cast<ConstantSDNode>(op->getOperand(1))->getZExtValue() !=
+        Intrinsic::set_loop_iterations) {
+        std::cerr << "Trying to lower invalid hwloop instruction" << std::endl;
+        return op;
+    }
+
+    // If hwloop is last instruction, let TDGen do the lowering
```

Comment:
.

Reply:
@pjaaskel  Missed the comment here?

Reply:
Meant that a full stop is missing at the end of the line.